src/main/java/sysc4806/project24/mini_survey_monkey/models/Question.java

```diff
@@ -0,0 +1,46 @@
+package sysc4806.project24.mini_survey_monkey.models;
+
+import jakarta.persistence.*;
+
+@Entity
+@Inheritance(strategy=InheritanceType.JOINED)
+public abstract class Question {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int questionId;
```

Comment:
**Could we change from `questionId` to `id`?**

Argument for change would be to eliminate redundant words.